web/babel.config.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   presets: [
-    ['@babel/preset-env', { modules: 'commonjs' }],
+    ['@babel/preset-env', { modules: 'auto', loose: true }],
```

Comment:
changed it to `commonjs` in [this issue](https://issues.tuerantuer.org/browse/WEBAPP-525). @Znippet would you mind checking out whether the webapp still compiles after this change?

Reply:
Using "auto" here is at least the "correct" way of doing this here :) That means webpack decides what it wants to have